fitstack/signal.py

```diff
@@ -957,9 +970,19 @@ def __init__(
         derivs: Optional[Dict[str, Tuple[float, float]]] = None,
         convolutions: Optional[Dict[str, Tuple[float, float]]] = None,
         kpar_range: Optional[Tuple[float, float]] = None,
+        nu21: float = 1420.40575177,  # MHz
+        z_eff: Optional[float] = None,  # effective redshift
+        cs: float = 299792.458,  # speed of light in km/s          
```

Comment:
```suggestion
        z_eff: Optional[float] = None,  # effective redshift     
```
`nu21` and the speed of light are defined in `cora.util.units`, so we should use those definitions.